### PR TITLE
Adjusting the translation of README.md

### PR DESCRIPTION
**Pull request by keyboard-slayer**
There was some translation mistakes on the  README. 
Some french expression doesn't translate well in english so I had to come with another way to express the idea.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,23 +35,23 @@ Join the [Cpcdos discord](https://discord.com/invite/3Qm8xDp)
 
 ## Official links
 - [Website - cpcdos.net](https://cpcdos.net)
-- [Forum - developpez.net](https://www.developpez.net/forums/f2044/systemes/autres-systemes/cpcdos)
+- [Forum - developpez.net](https://www.developpez.net/forums/f2044/systemes/autres-systemes/cpcdos) :warning: the forum is in french :warning:
 
 ## Update
-- From the [website](https://cpcdos.net) above.
+- From the [our website](https://cpcdos.net).
```
Comment (d0p1s4m4) [inline README.md]: I think `the` need to be removed

File: README.md [hunk 2]
```diff
@@ -2,31 +2,31 @@
...
- It is recommended to use [VMware workstation player 16](https://my.vmware.com/fr/web/vmware/downloads/details?downloadGroup=PLAYER-1610&productId=1039&rPId=55794) and not to have the pro version installed on your machine, as you may have less options to configure your network card.
-   Feel free to use an old cpcdos VM already configured to update it.
+ - Then simply run `OS2.1\_Make_CpcDos_.cwMake` _associated with CWC_ file
+ - Make sure you have an [editable] virtual machine (https://github.com/VLiance/VW_Viewer) or a bootable drive with Cpcdos installed (Cpcdos Tutorial [No1](https://www.youtube.com/user/cpcdososx) [No2](https://www.youtube.com/channel/UCkFCPxJF7ZzmWxW4i5WavCA/videos))
+ It is recommended to use [VMware workstation player 16](https://my.vmware.com/fr/web/vmware/downloads/details?downloadGroup=PLAYER-1610&productId=1039&rPId=55794) make sure that you don't have to pro version on your machine, as you may have less options to configure your network card.
```
Comment (d0p1s4m4) [inline README.md]: I think `to` (the ?) is a typo in `make sure that you don't have to pro version`

File: README.md (hunk 1 above)
Comment (keyboard-slayer) [reply README.md]: Totaly :eyes:

File: README.md (hunk 2 above)
Comment (keyboard-slayer) [reply README.md]: Yep sorry 😅

Comment (Keyboard-Slayer) [reply README.md:28]: Yep sorry 😅

**Review by d0p1s4m4 — APPROVED** [review]

**Pull request merged**